### PR TITLE
Some cleanup // Optimizers

### PR DESCRIPTION
**Pull request by Frightera**
This PR:
* Migrates docstrings into 4 indents 
* Removes unused args in adam & adamw

### CODE REVIEW

File: keras/optimizers/ftrl.py [hunk 1]
```diff
@@ -74,26 +74,27 @@ class Ftrl(optimizer.Optimizer):
...
-      l2_shrinkage_regularization_strength: A float value, must be greater than
-        or equal to zero. This differs from L2 above in that the L2 above is a
-        stabilization penalty, whereas this L2 shrinkage is a magnitude penalty.
-        When input is sparse shrinkage will only happen on the active weights.
-      beta: A float value, representing the beta value from the paper. Defaults
-        to 0.0.
-      {{base_optimizer_keyword_args}}
+        learning_rate: A `Tensor`, floating point value, a schedule that is a
+            `tf.keras.optimizers.schedules.LearningRateSchedule`, or a callable
+             that takes no arguments and returns the actual value to use. The
```
Comment (rchao) [inline keras/optimizers/ftrl.py:79]: Thanks for the PR! Looks like some of those lines don't have the proper indentation (indent 4) - can you fix this?

**Review by rchao — APPROVED** [review]
Thanks!

File: keras/optimizers/adagrad.py [hunk 2]
```diff
@@ -40,22 +40,22 @@ class Adagrad(optimizer.Optimizer):
...
+            either a floating point value,
+            or a `tf.keras.optimizers.schedules.LearningRateSchedule` instance.
+            Defaults to 0.001. Note that `Adagrad` tends to benefit from higher
+            initial learning rate values compared to other optimizers. To match
+            the exact form in the original paper, use 1.0.
+        initial_accumulator_value: Floating point value.
+            Starting value for the accumulators (per-parameter momentum values).
+            Must be non-negative.
+        epsilon: Small floating point value used to maintain numerical
+        stability.
```
Comment (haifeng-jin) [inline keras/optimizers/adagrad.py:53]: We need 4 more indents here.

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
Just one minor comment and we are ready to go. Thanks!

File: keras/optimizers/adagrad.py (hunk 2 above)
Comment (Frightera) [reply keras/optimizers/adagrad.py]: Ahh right.

**Review by haifeng-jin — APPROVED** [review]

**Pull request merged**